docker-compose.yml

```diff
@@ -2,11 +2,12 @@ version: "3.5"
 
 services:
   cardano-node:
-    image: inputoutput/cardano-node:1.35.3-configs
+    image: inputoutput/cardano-node:1.35.4
     environment:
       NETWORK:
+      CARDANO_NODE_SOCKET_PATH: ${CARDANO_NODE_SOCKET_PATH:-/ipc/node.socket}
     volumes:
-      - node-${NETWORK}-db:/data
+      - ./node-${NETWORK}-db:/data
```

Comment:
I understand the idea, however I believe we want to use volumes as opposed to bind mounts. The reasoning behind is to rely on docker machinery to manage the storage (security and the fact that no other process can mess up with the db). Important for enterprise users, like exchanges.

Reply:
Technically you still can mess it up, if you know where to look. 

But, ok I got the point. 👍